traversal/planner/GraphPlanner.java

```diff
@@ -278,11 +277,27 @@ private void optimise(GraphManager graphMgr, boolean singleUse) {
         printDebug(start, endSolver, end);
     }
 
-    private void updateOptimiserCoefficients() {
+    private void updateOptimiser() {
+        updateOptimiserCoefficents();
+        updateOptimiserConstraints();
+        if (!isVertexOrderInitialised) initialiseVertexOrderGreedy();
+        setOptimiserValues();
+        optimiser.reset();
```

Comment:
only one comment: it does look like we're resetting back to nothing - can we find a name that does reflect that we reset back to a feasible state or something?